npm/cypress-schematic/src/builders/cypress/schema.json

```diff
@@ -87,6 +92,10 @@
       "type": "boolean",
       "description": "Recompile and run tests when files change.",
       "default": false
+    },
+    "testingType": {
+      "type": "string",
+      "description": "Specify the type of tests to execute; either e2e or component. Defaults to e2e."
```

Comment:
We can add `"enum": ["e2e, "component"]`

Reply:
Done!